scenes/enemies/guard/guard.gd

```diff
@@ -383,8 +383,17 @@ func _notification(what: int) -> void:
 ## Function used for a tool button that either selects the current patrol_path
 ## in the editor, or creates a new one
 func edit_patrol_path() -> void:
+	if not Engine.is_editor_hint():
+		return
+
+	# Cannot directly reference [class EditorInterface] in code that isn't
+	# part of a script that runs only in the editor (like plugins).
+	# This function should only be called in the editor, but having a direct
+	# reference to the [class EditorInterface] causes errors on runtime builds.
+	var editor_interface = Engine.get_singleton("EditorInterface")
```

Comment:
An alternative to the early return `if not Engine.is_editor_hint()` would be:
```suggestion
	var editor_interface = Engine.get_singleton("EditorInterface")
	if not editor_interface:
		return
```
I guess both work.